beta-src/src/components/map/components/WDTerritory.tsx

```diff
@@ -175,6 +177,15 @@ const WDTerritory: React.FC<WDTerritoryProps> = function ({
     );
   };
 
+  const costalTerritories = [
+    "BULGARIA_NORTH_COAST",
+    "BULGARIA_SOUTH_COAST",
+    "SPAIN_NORTH_COAST",
+    "SPAIN_SOUTH_COAST",
+    "SAINT_PETERSBURG_NORTH_COAST",
+    "SAINT_PETERSBURG_SOUTH_COAST",
+  ];
```

Comment:
There is nothing in `territoryMapData` that Let's us know that this territory is a coastal. I don't have any other way to check. Any ideas?

Reply:
`territoryMapData` has a `type` field. At the moment we've only needed "land" and "water" types, but now we could use a "coast" type as well, so this can be added to the `TerritoryType` type. `type` in `territoryMapData` comes from `Territories.ts` so it can be updated to be coast there. The only other thing that would need to be done is see where the type is being used with "land" and make it useable with "coast" the same way.

Reply:
Another option would be to use the `TerritoryMap.ts` data and check whether the territory has a parent or not. Then you can save calculate once, save the list in redux, and reference it so that you know which territories to target with commands. This is probably the better option of the two.